cfy_manager/components/status_reporter/rabbitmq_status_reporter.py

```diff
@@ -30,3 +34,18 @@ def __init__(self, skip_installation):
         super(RabbitmqStatusReporter, self).__init__(skip_installation,
                                                      'rabbitmq_reporter',
                                                      BROKER_STATUS_REPORTER)
+
+    def configure(self):
+        super(RabbitmqStatusReporter, self).configure()
+        update_yaml_file(STATUS_REPORTER_CONFIGURATION_PATH,
```

Comment:
this call to update the reporter conf is written in several places, so util in status reporter